### ISSUE TITLE
Parser breaks when Postman folders has forward slash "/"

### ISSUE THREAD

**Issue by alittle-ninja** [appknox/postmanparser#16]
@subho007 @kirankumbhar The `get_items_of_folder` function from `collection.py` file split the folder name with a forward slash. This causes that if the folder name naturally has a slash or multiple of them, to break the parser with a `FolderNotFoundError` exception..

Since the folder names are normally HTTP endpoints, it is very frequently that this issues occurs.

**Comment by kirankumbhar** [comment]
The forward slash is used as a separator character. If the folder name has a slash it will assume each split string as a separate folder.  This is intended behavior similar to that of the file system on the operating system. One such e.g. is you cannot create a folder with the name "myfolder/test" on operating system. OS will assume `myfolder` as parent and `test` as a child inside it.

A solution will be to escape the forward slash of the `folder` argument value before passing it to `get_items_of_folder`.
Also, you will need to sanitize your collection schema to escape the forward slash in all folder names.

**Comment by alittle-ninja** [comment]
@kirankumbhar Yes, the problem it's that major companies uses the Postman folder names as endpoint separators, it is easy to fix when you are parsing your own folders, but when you are using third party collections, it becomes a problem. I will try to fix this localy, but the parser should ignore the slash in the folders, since the '/' is a valid character in postman folders, and this script is inteended to parse postman folders as well.

**Comment by kirankumbhar** [comment]
@alittle-ninja Ok. I got your point. Can you send one such example here? I'll try to accommodate this fix once I get some time.
